styles.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  max-width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  padding: 24px;
+  margin-top: 58px;
+}
+
+.nav-bar a {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.fa-bars {
+  background: #6070ff;
+}
```

Comment:
- [ ] Your Hamburger icon currently has a background color of blue , instead of the lines being blue as you can see below
> Your Design
![image](https://user-images.githubusercontent.com/90556221/170391122-db0e721c-0083-42ee-a062-26b6da015d26.png)

To change colors of font awesome icons, you might want to change the `color` property instead of the `background-color` property. Kindly implement these changes so it looks like the template